### ISSUE TITLE
product importer breaking products even after re-index

### ISSUE THREAD

**Issue by marcorosano** [dweeves/magmi-git#506]
I am having an issue on a site that I took over recently, where after the magmi importer updates the products, all the products that are in the csv file stop showing on the front end. 

If I don't re-index magento then the products show but the product url's display the 404 page.

After I re-index then all the products stop showing on the front end and I have to edit and save each one to get some of them to display, via bulk status change or individually changing any setting, this still doesn't work on some products though.

any help would be appriciated.

Thanks

**Comment by QuAzI** [comment]
Yep! I have the same issue. Product shown in administration pages, can be searched in category tree and through search page but have URL like https://<site>/catalog/product/view/id/21389/ and shows "404 Oops! Page not found".

Magento 1.14.2.4
